projects/kubernetes/kubernetes/1-26/patches/0001-IGNORE-THIS-PATCH.patch

```diff
@@ -0,0 +1,32 @@
+From cba455c311dc281180dd3854e7e30dfb9ca80a01 Mon Sep 17 00:00:00 2001
```

Comment:
Is this needed? I just tried running a build locally (didnt let it finish since it takes too long) but it seemed to get going. Im curious if you dug into to why/if this is still necessary?

Reply:
i see it actually, not sure why i didnt get it the first time, but ill open a pr